RealtimeCSG/Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/EditModeGUI/EditModeManager.cs

```diff
@@ -517,7 +517,7 @@ public static void OnSceneGUI(SceneView sceneView)
 					if (RealtimeCSG.CSGSettings.EnableRealtimeCSG)
 					{
 						// handle the tool
-						var sceneSize = sceneView.position.size;
+						var sceneSize = sceneView.rootVisualElement.contentRect;
```

Comment:
you might want to guard this with an ifdef for users of older unity versions, not sure when this was introduced ..

Reply:
this specific part was reverted back to master as it caused breaking changes as was mentioned by @HeadClot and @andreiagmu.